### ISSUE TITLE
2019-09-09T10:26:26.376-0700	ERROR	navigation/navigation.go:125	prowjobs.prow.k8s.io: crd conversion failed: cannot convert int64 to float64

### ISSUE THREAD

**Issue by jwalters-gpsw** [vmware/octant#247]
Starting octant using cluster with prow CRDs.

**Comment by jwalters-gpsw** [comment]
Full stack:

```
2019-09-09T10:26:25.764-0700	INFO	module/manager.go:75	registering action	{"component": "module-manager", "actionPath": "deployment/configuration", "module-name": "overview"}
2019-09-09T10:26:25.764-0700	INFO	api/content.go:52	Registering routes for overview
2019-09-09T10:26:25.765-0700	INFO	api/content.go:52	Registering routes for cluster-overview
2019-09-09T10:26:25.765-0700	INFO	api/content.go:52	Registering routes for configuration
2019-09-09T10:26:25.765-0700	INFO	dash/dash.go:335	Dashboard is available at http://127.0.0.1:53147

2019-09-09T10:26:26.376-0700	ERROR	navigation/navigation.go:125	prowjobs.prow.k8s.io: crd conversion failed: cannot convert int64 to float64
github.com/vmware/octant/pkg/navigation.CustomResourceDefinitions
	/go/src/github.com/vmware/octant/pkg/navigation/navigation.go:125
github.com/vmware/octant/pkg/navigation.CRDEntries
	/go/src/github.com/vmware/octant/pkg/navigation/navigation.go:74
github.com/vmware/octant/internal/octant.(*NavigationFactory).genNode
	/go/src/github.com/vmware/octant/internal/octant/factory.go:115
github.com/vmware/octant/internal/octant.(*NavigationFactory).Generate.func1
	/go/src/github.com/vmware/octant/internal/octant/factory.go:83
github.com/vmware/octant/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
	/go/src/github.com/vmware/octant/vendor/golang.org/x/sync/errgroup/errgroup.go:57
2019-09-09T10:26:26.404-0700	ERROR	navigation/navigation.go:125	prowjobs.prow.k8s.io: crd conversion failed: cannot convert int64 to float64
github.com/vmware/octant/pkg/navigation.CustomResourceDefinitions
	/go/src/github.com/vmware/octant/pkg/navigation/navigation.go:125
github.com/vmware/octant/pkg/navigation.CRDEntries
	/go/src/github.com/vmware/octant/pkg/navigation/navigation.go:74
github.com/vmware/octant/internal/octant.(*NavigationFactory).genNode
	/go/src/github.com/vmware/octant/internal/octant/factory.go:115
github.com/vmware/octant/internal/octant.(*NavigationFactory).Generate.func1
	/go/src/github.com/vmware/octant/internal/octant/factory.go:83
github.com/vmware/octant/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
	/go/src/github.com/vmware/octant/vendor/golang.org/x/sync/errgroup/errgroup.go:57
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1e16d69]

goroutine 260 [running]:
github.com/vmware/octant/pkg/navigation.CRDEntries.func1(0x0, 0x18, 0x1)
	/go/src/github.com/vmware/octant/pkg/navigation/navigation.go:80 +0x49
sort.doPivot_func(0xc00090bd38, 0xc000202220, 0x0, 0x1d, 0xc001028540, 0x8)
	/usr/local/go/src/sort/zfuncversion.go:89 +0x416
sort.quickSort_func(0xc00090bd38, 0xc000202220, 0x0, 0x1d, 0xa)
	/usr/local/go/src/sort/zfuncversion.go:143 +0x9a
sort.Slice(0x211d400, 0xc0002021c0, 0xc00090bd38)
	/usr/local/go/src/sort/slice.go:21 +0x129
github.com/vmware/octant/pkg/navigation.CRDEntries(0x29b8620, 0xc0012503c0, 0xc001133940, 0x34, 0xc0011b8567, 0x7, 0x29cd260, 0xc0000c02a0, 0x0, 0x0, ...)
	/go/src/github.com/vmware/octant/pkg/navigation/navigation.go:79 +0x124
github.com/vmware/octant/internal/octant.(*NavigationFactory).genNode(0xc0000c0e40, 0x29b8620, 0xc0012503c0, 0x23d9308, 0x10, 0x27d7158, 0xc0006400d0, 0xc001072f60, 0x23d5f9c)
	/go/src/github.com/vmware/octant/internal/octant/factory.go:115 +0x269
github.com/vmware/octant/internal/octant.(*NavigationFactory).Generate.func1(0xc000000008, 0x27da070)
	/go/src/github.com/vmware/octant/internal/octant/factory.go:83 +0xeb
github.com/vmware/octant/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1(0xc001073020, 0xc0003f3980)
	/go/src/github.com/vmware/octant/vendor/golang.org/x/sync/errgroup/errgroup.go:57 +0x57
created by github.com/vmware/octant/vendor/golang.org/x/sync/errgroup.(*Group).Go
	/go/src/github.com/vmware/octant/vendor/golang.org/x/sync/errgroup/errgroup.go:54 +0x66
exit status 2

```

**Comment by jwalters-gpsw** [comment]
Version:
```
Version:  0.5.1
Git commit:  e0397d2596a7575abf4ca9aeacafd92912d484e0
Built:  2019-08-06T19:40:35Z

```

**Comment by jwalters-gpsw** [comment]
Comment in code seems to suggest there might be a fix related to this merge:
https://github.com/kubernetes-sigs/yaml/pull/14

**Comment by wwitzel3** [comment]
Hi @jwalters-gpsw , a fix for this issue was merged and released in our 0.6 release (#145).
Could you try downloading and using the latest Octant.

https://github.com/vmware/octant/releases/latest

**Comment by jwalters-gpsw** [comment]
Looks good
